portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -22,11 +31,83 @@
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
-public class DataServlet extends HttpServlet {
+public class DataServlet extends HttpServlet{
+    /**
+    * set of constants that are POST request parameter keys and Entity property keys
+    */
+    private static final String CONTACT = "Contact";
+    private static final String FIRST_NAME = "firstname";
+    private static final String LAST_NAME = "lastname";
+    private static final String EMAIL = "email";
+    private static final String SUBJECT = "subject";
+    private static final String TIMESTAMP = "timestamp";
+    private static final Gson gson = new Gson();
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        Query query = new Query(CONTACT);
```

Comment:
is this a contact of a comment?

Reply:
the comments are displaying contact info in my form just for the sake of display